Integrations/SlashNextPhishingIncidentResponse/SlashNextPhishingIncidentResponse.py

```diff
@@ -0,0 +1,1180 @@
+    }
+    response = http_request(endpoint=HOST_REPORT_API, data=api_data)
+
+    if response.get('errorNo') != 0:
+        return_error('API Returned, {}:{}!'.format(response.get('errorNo'), response.get('errorMsg')))
+
+    return response
+
+
+def host_urls_command():
+    """
+    Execute SlashNext's host/report API against the requested host urls command with the given parameters
+    @:return: None
+    """
+    # 1. Get input host and limit from Demisto
+    host = demisto.args().get('host')
+    limit = demisto.args().get('limit')
+    # 2. Get the host report from SlashNext API
+    response = host_urls(host=host, limit=limit)
+    if response.get('errorNo') != 0:
+        return
+    # 3. Parse and format the response
+    snx_ioc_cont_list = []       # type: List[Dict[str, str]]
+    dbot_score_cont_list = []    # type: List[Dict[str, str]]
+    url_cont_list = []           # type: List[Dict[str, str]]
+    snx_ec_cont_list = []        # type: List[Dict[str, str]]
+    for url_data in response.get('urlDataList'):
+        if url_data.get('threatData').get('verdict').startswith('Unrated') is False:
+            snx_ioc_cont, dbot_score_cont, url_cont = get_snx_url_ioc_context(url_data, is_scan=True)
+            snx_ioc_cont_list.extend(snx_ioc_cont)
+            dbot_score_cont_list.extend(dbot_score_cont)
+            url_cont_list.extend(url_cont)
+            snx_ec_cont_list.append(snx_ioc_cont[0])
+
+    ec = {}    # type: Dict[str, List[Dict[str, str]]]
+    if response.get('urlDataList')[0].get('threatData').get('verdict').startswith('Unrated') is False:
+        ec = {
+            'SlashNext.URL(val.Value === obj.Value)': snx_ec_cont_list,
+            'DBotScore': dbot_score_cont_list
+        }
+        if len(url_cont_list) != 0:
+            ec['URL'] = url_cont_list
+
+    host = host.encode('idna')
+
+    title = 'SlashNext Phishing Incident Response - Host URLs\n' \
+            '##### host = {}'.format(host.decode())
+
+    if response.get('normalizeData').get('normalizeStatus') == 1:
+        title += ' *\n*' + response.get('normalizeData').get('normalizeMessage')
+
+    md = tableToMarkdown(
+        title,
+        snx_ioc_cont_list,
+        ['Value',
+         'Type',
+         'Verdict',
+         'ScanID',
+         'ThreatStatus',
+         'ThreatName',
+         'ThreatType',
+         'FirstSeen',
+         'LastSeen']
+    )
+
+    return_outputs(md, ec, snx_ioc_cont_list)
+
+
+def url_scan(url):
+    """
+    Execute SlashNext's url/scan API against the requested URL scan with the given parameters
+    :param url: URL to be scanned
+    :return: Response of the SlashNext url/scan API
+    """
+    # Create the required data dictionary for URL/Scan
+    api_data = {
+        'url': url
+    }
+    response = http_request(endpoint=URL_SCAN_API, data=api_data)
+
+    if response.get('errorNo') == 1:
+        ec = {
+            'SlashNext.URL.ScanID': response.get('urlData').get('scanId')
```

Comment:
Also return here the `URL`
Need to use here DT, otherwise running twice will create two entries in the context